doc/dnf5_plugins/builddep.8.rst

```diff
@@ -0,0 +1,73 @@
+..
+    Copyright Contributors to the libdnf project.
+
+    This file is part of libdnf: https://github.com/rpm-software-management/libdnf/
+
+    Libdnf is free software: you can redistribute it and/or modify
+    it under the terms of the GNU General Public License as published by
+    the Free Software Foundation, either version 2 of the License, or
+    (at your option) any later version.
+
+    Libdnf is distributed in the hope that it will be useful,
+    but WITHOUT ANY WARRANTY; without even the implied warranty of
+    MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+    GNU General Public License for more details.
+
+    You should have received a copy of the GNU General Public License
+    along with libdnf.  If not, see <https://www.gnu.org/licenses/>.
+
+.. _builddep_plugin_ref-label:
+
+#################
+ Builddep Command
+#################
+
+Synopsis
+========
+
+``dnf5 builddep [options] [<package>...]``
+
+
+Description
+===========
+
+Install missing dependencies for building an RPM package.
+
+.. warning:: Build dependencies in a source package (i.e. src.rpm) might be different
+             than you would expect because they were evaluated according macros
+             set on the package build host.
+
+Options
+=======
+
+``--allowerasing``
+    Allow erasing of installed packages to resolve dependencies resolution problems.
+
+``--skip-unavailable``
+    Allow skipping build dependencies not available in repositories. All available build dependencies will be installed.
+
+``-D "macro expr", --define="macro expr"``
+    Define the RPM macro named "macro" to the value "expr" when parsing spec files. Does not apply for source rpm files.
```

Comment:
I would rephrase this like 'Define a rpm macro. Set the value "expr" to the macro "macro" when parsing spec files. ...'
or something a written in a shorter form than what's in the committed file. what do you think?

Reply:
Changed.